ui-tests/src/test/java/io/syndesis/qe/pages/customizations/extensions/TechExtensionsListComponent.java

```diff
@@ -0,0 +1,70 @@
+package io.syndesis.qe.pages.customizations.extensions;
+
+import static com.codeborne.selenide.Condition.exactText;
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.$$;
+
+import java.util.Iterator;
+
+import org.openqa.selenium.By;
+
+import com.codeborne.selenide.ElementsCollection;
+import com.codeborne.selenide.SelenideElement;
+
+import io.syndesis.qe.pages.SyndesisPageObject;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class TechExtensionsListComponent extends SyndesisPageObject {
+
+	private static final class Element {
+		public static final By ROOT = By.cssSelector("syndesis-tech-extensions-list");
+
+		public static final By ITEM = By.className("list-pf-item");
+		public static final By ITEM_TITLE = By.className("list-pf-title");
+		public static final By LIST_WRAPPER = By.cssSelector("pfng-list");
+	}
+
+	@Override
+	public SelenideElement getRootElement() {
+		return $(Element.ROOT).shouldBe(visible);
+	}
+
+	@Override
+	public boolean validate() {
+		return $(Element.ROOT).is(visible);
+	}
+
+	public SelenideElement getExtensionItem(String name) {
+		$(Element.LIST_WRAPPER).shouldBe(visible);
+		
+		ElementsCollection items = $$(Element.ITEM);
+		
+		for (Iterator<SelenideElement> iterator = items.iterator(); iterator.hasNext();) {
```

Comment:
You might want to look into Java 8 filters. It would look a bit nicer :)

Reply:
OK, i will take a look.